README.md

```diff
@@ -69,6 +73,24 @@ An example of `--node-label-expr`:
 (metadata.labels.region == 'us-west-1' && metadata.labels.app == 'nginx') || (metadata.labels.region == 'us-west-2' && metadata.labels.app == 'nginx')
 ```
 
+### Ignore pod controlled by ...
+It is possible to prevent eviction of pods that are under control of:
+- daemonset
+- statefulset
+- Custom Resource
+- ...
+
+or not even on control of anything. For this, use the flag `do-not-evict-pod-controlled-by`; it can be repeated. An empty value means that we block eviction on pods that are uncontrolled.
+The value can be a `kind` or a `kind.group` or a `kind.version.group` to designate the owner resource type. If the `version` or/and the `group` are omitted it acts as a wildcard (any version, any group). It is case-sensitive and must match the API Resource definition.
```

Comment:
A link to https://godoc.org/k8s.io/apimachinery/pkg/runtime/schema#ParseKindArg might be helpful here as well to provide a reference "forthis.v1.syntax."